ci-operator/config/openshift/ci-tools/openshift-ci-tools-master.yaml

```diff
@@ -408,6 +412,10 @@ images:
       paths:
       - destination_dir: boskos/
         source_path: /app
+    prometheus:
```

Comment:
Can we maybe just add this to the `build_root` so we don't have to do build this image on every testrun?

Reply:
Sure, I will add the `boskos` and `prometheus` binaries to the root